internal/controller/dbinstance_controller.go

```diff
@@ -191,10 +197,48 @@ func (r *DbInstanceReconciler) create(ctx context.Context, dbin *kindav1beta1.Db
 
 		instance = dbinstance.GsqlNew(name, config, user, password, apiEndpoint)
 	case "generic":
+		var host string
+		var port uint16
+		var publicIP string
+
+		from := dbin.Spec.Generic.HostFrom
```

Comment:
is this assignment needed? can you not just test direct this in the if condition?

Reply:
Should be fixed